### ISSUE TITLE
Multiline events with blank lines missing in Splunk

### ISSUE THREAD

**Issue by epernice** [splunk/splunk-connect-for-kubernetes#674]
**What happened**:

Multiline events containing blank lines do not show up in Splunk

**What you expected to happen**:

Multiline events with blank lines should show up in Splunk

**How to reproduce it (as minimally and precisely as possible)**:

Configure multiline events:

```
splunk-kubernetes-logging:
  logs:
    default:
      from:
        pod: "*"
        container: "*"
      multiline:
        firstline: /\d{4}\-\d{2}\-\d{2} \d{2}\:\d{2}\:\d{2}[\.\,]\d{3}/
```

Log a multiline event with a blank line, such as:

```
2021-11-17 19:12:28.398 Line 1
Line 2

Line 3
```

The event does not show up in Splunk

**Anything else we need to know?**:

Events without blank lines show up correctly in Splunk. For example:

```
2021-11-17 19:12:28.398 Line 1
Line 2
Line 3
```

**Environment**:
- Kubernetes version (use `kubectl version`):

Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-18T16:12:00Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.6", GitCommit:"8a62859e515889f07e3e3be6a1080413f17cf2c3", GitTreeState:"clean", BuildDate:"2021-04-15T03:19:55Z", GoVersion:"go1.15.10", Compiler:"gc", Platform:"linux/amd64"}

- Ruby version (use `ruby --version`):
- OS (e.g: `cat /etc/os-release`):

NAME="Oracle Linux Server"
VERSION="7.9"
ID="ol"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.9"
PRETTY_NAME="Oracle Linux Server 7.9"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:oracle:linux:7:9:server"
HOME_URL="https://linux.oracle.com/"
BUG_REPORT_URL="https://bugzilla.oracle.com/"

ORACLE_BUGZILLA_PRODUCT="Oracle Linux 7"
ORACLE_BUGZILLA_PRODUCT_VERSION=7.9
ORACLE_SUPPORT_PRODUCT="Oracle Linux"
ORACLE_SUPPORT_PRODUCT_VERSION=7.9

- Splunk version: 8.2.2 Build 87344edfcdb4
- Splunk Connect for Kubernetes helm chart version: splunk-connect-for-kubernetes-1.4.9

**Comment by foram-splunk** [comment]
Hi @epernice ,

The same issue was raised #597 , which is resolved in the 1.4.10 release .

PR for the same : #664 .

Let us know if you're still facing the issue . Thanks!

**Comment by epernice** [comment]
Updating to 1.4.10 resolved the issue - thanks!
